tests/sessions/claims/test_set_claim_value.py

```diff
@@ -68,16 +79,16 @@ async def test_should_overwrite_claim_value_using_session_handle(timestamp: int)
     s = await create_new_session(dummy_req, "someId")
 
     payload = s.get_access_token_payload()
-    assert payload == {"st-true": {"t": timestamp, "v": True}}
+    assert len(payload) == 8
+    assert payload["st-true"] == {"t": timestamp, "v": True}
 
     await set_claim_value(s.get_handle(), TrueClaim, False)
 
-    # Payload should be updated now:
-    # Note that the session var (s) still contains the old payload.
-    # We need to fetch the new one. so we will use get_session_information()
+    # Check after update:
     s = await get_session_information(s.get_handle())
     assert s is not None
     payload = s.access_token_payload
+    # FIXME: Shouldn't this have other protected_keys as well??
```

Comment:
fixme

Reply:
It's actually about the get_session_information vs session.access_token_payload difference that we discussed. What did we decide to do about that inconsistency?

Reply:
the access token payload in get_session_information has been renamed.

Reply:
Done. Renamed in python as well.